### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by praveenrewar**
Fixes https://github.com/kubernetes/sig-security/issues/36

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Vim-related files
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+
+# User cluster configs
+.kubeconfig
```
Comment (Matheusafonsouza) [inline .gitignore:32]: Hey! I think that's better add the other files in[this](https://github.com/kubernetes/kubernetes/blob/master/.gitignore) gitignore, can be useful in future.

**Review by Matheusafonsouza — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (praveenrewar) [reply .gitignore:32]: If these are going to be required in the future, then sure, I will add them :)

Comment (PushkarJ) [reply .gitignore:32]: @Matheusafonsouza Most of those files in k/k are repo specific. So agree with @praveenrewar that we could add them later in a separate PR. The current file is a good starting point.

**Comment by PushkarJ** [comment]
/lgtm

Thanks @praveenrewar for working on this and congrats on your first contribution to `k/sig-security`

File: .gitignore (hunk 1 above)
Comment (Matheusafonsouza) [reply .gitignore:32]: @PushkarJ Okay! Thanks :)

**Comment by tabbysable** [comment]
/approve

Thanks @praveenrewar

**Pull request merged**